src/util/reconstructor.rs

```diff
@@ -90,6 +90,7 @@ impl Reconstructor {
             "file" => {
                 Decoder::new_from_file(Path::new(&(directory.clone() + "/" + &aedat_filename_0)))?
             }
+            #[cfg(target_family = "unix")]
```

Comment:
Set up a GitHub Actions pipeline on your fork to test it for Linux & Windows before we merge. Use [this](https://github.com/ac-freeman/adder-codec-rs/blob/main/.github/workflows/rust_action_test_and_docs.yml) for inspiration.